Narre restart

BinbinZhou-NOAA opened the pull request:
**Pull Request Testing Instruction:** 

**Part I:  The narre stat job restart testing procedures**

 1. ✔️Scenario 1, first run without restart (all sub-tasks can be completed without interruption)
    step (1):
          in ../EVS/dev/drivers/scripts/stats/narre directory, run stat driver script
           qsub jevs_narre_stats.sh

    After the job is completed, then
      step (2) check the output stat file in the $COMOUTfinal, it should contain the output stat file file
                evs.stats.narremean.mean.grid2obs.v${VDATE}.stat

           (3) check output files for restart in $COMOUTsmall, should contain the final stat files:
              a. small stat files
              b. prepbufr.${VDATE} directory which contains 16 prepared prepbufr nc files
              c. 8 *.completed files
                 run_narre_narremean.Alaska.range1.completed
                 run_narre_narremean.Alaska.range2.completed
                 run_narre_narremean.Alaska.range3.completed
                 run_narre_narremean.Alaska.range4.completed
                 run_narre_narremean.CONUS.range1.completed
                 run_narre_narremean.CONUS.range2.completed
                 run_narre_narremean.CONUS.range3.completed
                 run_narre_narremean.CONUS.range4.completed
              These *.completed files  indicate the all of the 8 stat sub-tasks have been completed

 2. ✔️ Scenario 2: if prepare procedure was completed, but all 8 stat sub-tasks were not completed
     In this case, the prepbufr.${VDATE} directory exists in $COMOUTsmall, but no *.completed files
                   Maybe there are a part of of stat files or no any stat files in $COMOUTsmall
                   and no stat file in the $COMOUTfinal directory
     To simulate this scenario,
       step (1)delete all *.completed in $COMOUTsmall
                 and delete the final stat file in $$COMOUTfinal
               Just keep the prepbufr.${VDATE} directory in $COMOUTsmall
       step (2) run the stat driver script in ../EVS/dev/drivers/scripts/stats/narre directory
                qsub jevs_narre_stats.sh
       step (3) After the stat job is completed, check $COMOUTfinal, it should contain the final stat file
               evs.stats.narremean.mean.grid2obs.v${VDATE}.stat
               if you like, check the  $COMOUTsmall, and see all 8 *.completed files

3. ✔️ Scenario 3 if prepare procedure was completed, and only a part of stat sub-task were completed
      In this case, the prepbufr.${VDATE} directory exists in $COMOUTsmall, and only *.completed files exist
        for those completed sub-tasks.
      To simulate this scenario, suppose all Alaska sub-tasks have been completed, and all CONUS sub-tasks were not
       step (1) delete  run_narre_narremean.CONUS.*.completed (4 files) from $COMOUTsmall and
                delete the final stat file in $COMOUTfinal
       step (2) run the stat driver script in ../EVS/dev/drivers/scripts/stats/narre directory
                qsub jevs_narre_stats.sh
       step (3) After the stat job is completed, check $COMOUTfinal, it should contain the final stat file
               evs.stats.narremean.mean.grid2obs.v${VDATE}.stat
               if you like, check the  $COMOUTsmall, and see all 8 *.completed files

 4. ✔️ Scenario 4: if prepare procedure and all stat sub-tasks were  completed, but no final stat file in $COMOUTfinal
      This means, the gather procedure was not completed.
      In this case, prepbufr.${VDATE} directory and all of 8 *completed files exist in $COMOUTsmall,
        but final stat file evs.stats.narremean.mean.grid2obs.v${VDATE}.stat is missing from $COMOUTfinal
      To simulate this scenario,
         step (1) just delete the file evs.stats.narremean.mean.grid2obs.v${VDATE}.stat
                  from $COMOUTfinal
         step (2) run the stat driver script in ../EVS/dev/drivers/scripts/stats/narre directory
                qsub jevs_narre_stats.sh
         step (3) After the stat job is completed, check $COMOUTfinal, it should contain the final stat file
               evs.stats.narremean.mean.grid2obs.v${VDATE}.stat
               if you like, check the  $COMOUTsmall, and see all 8 *.completed files
  5. ✔️Scenario 5: prepare and sub-stat tasks were not completed.
         In this case, prepbufr.${VDATE} directory and all of 8 *completed files are not existing in $COMOUTsmall
      To simulate this scenario,
          step (1) delete all of files in $COMOUTsmall and $$COMOUTfinal
          then repeat the steps of scenario 1.

**Part II:  The narre plot job restart testing procedures**

  1. ✔️Scenario 1, first run without restart (all sub-tasks can be completed without interruption)
      step (1):
          in ../EVS/dev/drivers/scripts/plots/narre directory, run stat driver script
           qsub jevs_narre_past31days_plots.sh  (or qsub jevs_narre_past90days_plots.sh)

      step (2): check output files
           If all of the sub-tasks were completed, the final tar file
             evs.plots.narre.grid2obs.last31days.v${VDATE}.tar or
             evs.plots.narre.grid2obs.last90days.v${VDATE}.tar
           should be in $COMOUT directory. And there is another directory called "restart" also
            exists in $COMOUT
           In the restart directory, there is another directory "31" or "90", which contain
             4 performance*.png files and 4 *.completed files

  2. ✔️Scenario 2: if only some of the sub-tasks were completed and all other png files were not generated
     in this case, the final tar file could exist or not, it is not completed and all png files that were
        not generated are missing from the final tar files. Then need restart
     To simulate this case, suppose run 31-day job, and suppose all ceiling plots were generated while all vis plot not
      step (1): delete all performance*vis*.png and all run_narre*VISsfc*.completed files from  $COMOUT/restart/31
                delete the final tar file from $COMOUT
      step (2): run jevs_narre_past31days_plots.sh in the driver directory ../EVS/dev/drivers/scripts/plots/narre directory
      step (3): if the job completed, check output files to see all of 4 *.completed files exist in
                $COMOUT/restart/31 and the final tar file in $OMOUT

  3. ✔️Repeat scenario 1 and 2 for 90-day job in the same steps

BinbinZhou-NOAA committed:
Add restart capability

BinbinZhou-NOAA committed:
Add restart capability to narre plotting script

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into narre_restart

ShelleyMelchior-NOAA commented:
Great instructions, @BinbinZhou-NOAA ! Thank you!

BinbinZhou-NOAA committed:
update J-jobs for restart

BinbinZhou-NOAA commented:
Shelley, 

  The "mkdir directory" commands in J-jobs were updated by  checking the  directory existing status for restart

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into narre_restart

ShelleyMelchior-NOAA commented:
Hi @BinbinZhou-NOAA , I have cloned your fork and checked out branch `narre_restart`.

BinbinZhou-NOAA commented:
Shelley,

    If you have any questions during the testing, please let me know.

Thanks!

Binbin


On Tue, Apr 16, 2024 at 11:05 AM Shelley Melchior ***@***.***>
wrote:

> Hi @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , I have cloned
> your fork and checked out branch narre_restart.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2059316957>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFIHQFUIBXZ32NWQ6LDY5U43LAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANJZGMYTMOJVG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
**Part 1, Scenario 1:** ✔️ 

logfile: `/lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142562226`

COMOUTfinal:
```
$ ls -l /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240415/
total 2516
-rw-r--r-- 1 shelley.melchior vpppg 2576142 Apr 16 16:24 evs.stats.narremean.mean.grid2obs.v20240415.stat
```
COMOUTsmall=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/atmos.20240415/narre/grid2obs`
Contains 192 small stat files.
Contains the  `prepbufr.20240415/` directory with 17 nc files.
Contains 8 empty run status `*.completed` files.

Once you confirm these results are expected, I will move onto Part 1, Scenario 2.

BinbinZhou-NOAA commented:
Shelley,

  I can confirm that the output stat file is good, and all files for
restart in the directory
/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/
v1.0/stats/narre/atmos.20240415
are good. So you can go ahead to the next scenario testing.

Thanks!

Binbin

On Tue, Apr 16, 2024 at 12:34 PM Shelley Melchior ***@***.***>
wrote:

> *Part 1, Scenario 1:*
>
> logfile: /lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142562226
>
> COMOUTfinal:
>
> $ ls -l /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240415/
> total 2516
> -rw-r--r-- 1 shelley.melchior vpppg 2576142 Apr 16 16:24 evs.stats.narremean.mean.grid2obs.v20240415.stat
>
> COMOUTsmall=
> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/atmos.20240415/narre/grid2obs
> Contains 192 small stat files.
> Contains the prepbufr.20240415/ directory with 17 nc files.
> Contains 8 empty run status *.completed files.
>
> Once you confirm these results are expected, I will move onto Part 1,
> Scenario 2.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2059501006>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFL3ME3J5KR2LUP3LNDY5VHKRAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANJZGUYDCMBQGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
**Part 1, Scenario 2:** ✔️ 

I removed the `*.completed` files in COMOUTsmall.
I removed the stat file in COMOUTfinal.

logfile: `/lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142570705`

COMOUTfinal: (note, the following result matches that in Part1, Scenarion 1 ✔️ )
```
$ ls -l /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240415
total 2516
-rw-r--r-- 1 shelley.melchior vpppg 2576142 Apr 16 17:47 evs.stats.narremean.mean.grid2obs.v20240415.stat
```

COMOUTsmall=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/atmos.20240415/narre/grid2obs`
The 192 *.stat files were re-written with a later time stamp.
There are 8 empty status `*.completed` files.
The nc files in the `prepbufr.20240415/` dir have the older time stamp. These were retained for this scenario.

Please confirm these expected results.

BinbinZhou-NOAA commented:
Shelley,

 The existing prepbufr files in $COMOUTsmall are correctly copied to the
WORKING directory.
 The final stat file in $COMOUTfinal is good, and all files in $COMOUTsmall
are good as well.

Thanks!

Binbin

On Tue, Apr 16, 2024 at 1:56 PM Shelley Melchior ***@***.***>
wrote:

> *Part1, Scenario 2:*
>
> I removed the *.completed files in COMOUTsmall.
> I removed the stat file in COMOUTfinal.
>
> logfile: /lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142570705
>
> COMOUTfinal: (note, the following result matches that in Part1, Scenarion
> 1 ✔️ )
>
> $ ls -l /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240415
> total 2516
> -rw-r--r-- 1 shelley.melchior vpppg 2576142 Apr 16 17:47 evs.stats.narremean.mean.grid2obs.v20240415.stat
>
> COMOUTsmall=
> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/atmos.20240415/narre/grid2obs
> The 192 *.stat files were re-written with a later time stamp.
> There are 8 empty status *.completed files.
> The nc files in the prepbufr.20240415/ dir have the older time stamp.
> These were retained for this scenario.
>
> Please confirm these expected results.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2059650292>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFLAKUW75Q5JXYWKOU3Y5VQ6BAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANJZGY2TAMRZGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Thanks Binbin!

**Part 1, Scenario 3**
I deleted the CONUS `*.completed files` from COMOUTsmall.
I deleted the final stats file from COMOUTfinal.

logfile: `/lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142582628`

COMOUTfinal: (note, the following result matches those in Part1, Scenarios 1 ✔️  and 2 ✔️ )
```
$ ls -l /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240415
total 2516
-rw-r--r-- 1 shelley.melchior vpppg 2576142 Apr 16 19:52 evs.stats.narremean.mean.grid2obs.v20240415.stat
```

COMOUTsmall=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/atmos.20240415/narre/grid2obs`
The 192 `*.stat` files were re-written with a later time stamp.
The CONUS `*.completed` files (4) were re-written w/ a later time stamp.
The nc files in the `prepbufr.20240415/` dir have the older time stamp. These were retained for this scenario.

Please confirm these expected results.

BinbinZhou-NOAA commented:
Shelley,

  In scenario 3,  suppose Alaska stats are completed but CONUS stats are
not.
So just delete all *CONUS*.completed and keep all *Alaska*.completed, and
delete all CONUS*stat files from the  COMOUTsmall directory, and delete
final
stat file from COMfinal directory.
Then re-run the driver script

Thanks!

Binbin

On Tue, Apr 16, 2024 at 4:22 PM Shelley Melchior ***@***.***>
wrote:

> Thanks Binbin!
>
> *Part 1, Scenario 3*
> I deleted the CONUS *.completed files from COMOUTsmall.
> I deleted the final stats file from COMOUTfinal.
>
> logfile: /lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142582628
>
> COMOUTfinal: (note, the following result matches those in Part1, Scenarios
> 1 ✔️ and 2 ✔️ )
>
> $ ls -l /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240415
> total 2516
> -rw-r--r-- 1 shelley.melchior vpppg 2576142 Apr 16 19:52 evs.stats.narremean.mean.grid2obs.v20240415.stat
>
> COMOUTsmall=
> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/atmos.20240415/narre/grid2obs
> The 192 *.stat files were re-written with a later time stamp.
> The CONUS *.completed files (4) were re-written w/ a later time stamp.
> The nc files in the prepbufr.20240415/ dir have the older time stamp.
> These were retained for this scenario.
>
> Please confirm these expected results.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2059861591>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFLK7C32I3LMUYWNB5TY5WB6FAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANJZHA3DCNJZGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
> Shelley, In scenario 3, suppose Alaska stats are completed but CONUS stats are not. So just delete all *CONUS*.completed and keep all *Alaska*.completed, and delete all CONUS*stat files from the COMOUTsmall directory, and delete final stat file from COMfinal directory. Then re-run the driver script Thanks! Binbin

Ah, ok! Thanks, Binbin. I think I didn't see in your instructions that I needed to delete the CONUS stats files from COMOUTsmall. I will make that adjustment and re-run.

ShelleyMelchior-NOAA commented:
**Part 1, Scenario 3** TAKE 2!
I deleted the CONUS `*.completed files` from COMOUTsmall.
I deleted the CONUS stat files from COMOUTsmall.
I deleted the final stats file from COMOUTfinal.

logfile: `/lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142604578`

COMOUTfinal: (note, the following result matches those in Part1, Scenarios 1 ✔️  and 2 ✔️ )
```
$ ls -l /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240415
total 2516
-rw-r--r-- 1 shelley.melchior vpppg 2576142 Apr 17 00:29 evs.stats.narremean.mean.grid2obs.v20240415.stat
```

COMOUTsmall=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/atmos.20240415/narre/grid2obs`
The 96 CONUS `*.stat` files were re-written with a later time stamp.
The CONUS `*.completed` files (4) were re-written w/ a later time stamp.
The nc files in the `prepbufr.20240415/` dir have the older time stamp. These were retained for this scenario.

Please confirm these expected results.

BinbinZhou-NOAA commented:
Shelley,

  The final stat file is good, and all files in the COMOUTsmall directory
are good as well.

BTW, in your previous email, you deleted all *.completed files and final
stat file.
That test is actually for scenario 5.  In that testing, all results are
good.
So right now scenario 1,2,3, and 5 are all passed.

Thanks!

Binbin

On Tue, Apr 16, 2024 at 9:03 PM Shelley Melchior ***@***.***>
wrote:

> *Part 1, Scenario 3* TAKE 2!
> I deleted the CONUS *.completed files from COMOUTsmall.
> I deleted the CONUS stat files from COMOUTsmall.
> I deleted the final stats file from COMOUTfinal.
>
> logfile: /lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142604578
>
> COMOUTfinal: (note, the following result matches those in Part1, Scenarios
> 1 ✔️ and 2 ✔️ )
>
> $ ls -l /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240415
> total 2516
> -rw-r--r-- 1 shelley.melchior vpppg 2576142 Apr 17 00:29 evs.stats.narremean.mean.grid2obs.v20240415.stat
>
> COMOUTsmall=
> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/atmos.20240415/narre/grid2obs
> The 96 CONUS *.stat files were re-written with a later time stamp.
> The CONUS *.completed files (4) were re-written w/ a later time stamp.
> The nc files in the prepbufr.20240415/ dir have the older time stamp.
> These were retained for this scenario.
>
> Please confirm these expected results.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2060155474>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFNNJ37LJ7KFEZ5RELTY5XC6XAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRQGE2TKNBXGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Thanks Binbin! I don't think I actually did what you suggest, at least not intentionally. What happened is when I did the re-run for Part 1, Scenario 3 last night it was late enough that the VDATE had cycled over to 20240416. So that run actually was from scratch and ran through like normal. I went into the driver and forced VDATE=20240415 and ran it again in order to replicate Part 1, Scenario 3 properly. For COMOUTsmall, you will see the timestamps on the CONUS files are a lot later. The timestamps for the Alaska files are the original timestamps. I never deleted those. To be sure, I will intentionally test Part 1, Scenario 5! But first, I will move onto Part 1, Scenario 4 next.

BinbinZhou-NOAA commented:
Shelley,

   I just noticed  that the timestamps of  Alaska and CONUS stat/completed
files are different in $COMOUTsmall.
Thanks for the detailed testing procedures.  I'll check the output of
Scenario 4 and 5 after you
completed them.

Binbin

On Wed, Apr 17, 2024 at 10:23 AM Shelley Melchior ***@***.***>
wrote:

> Thanks Binbin! I don't think I actually did what you suggest, at least not
> intentionally. What happened is when I did the re-run for Part 1, Scenario
> 3 last night it was late enough that the VDATE had cycled over to 20240416.
> So that run actually was from scratch and ran through like normal. I went
> into the driver and forced VDATE=20240415 and ran it again in order to
> replicate Part 1, Scenario 3 properly. For COMOUTsmall, you will see the
> timestamps on the CONUS files are a lot later. The timestamps for the
> Alaska files are the original timestamps. I never deleted those. To be
> sure, I will intentionally test Part 1, Scenario 5! But first, I will move
> onto Part 1, Scenario 4 next.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2061384709>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFLHW34IWYZC62QE4ZDY52AU3AVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRRGM4DINZQHE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
**Part 1, Scenario 4**

For this test I am using VDATE=20240416.
I deleted the final stat file in COMOUTfinal.

logfile: `/lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142666347`

The final stat file was rewritten (later time stamp).
```
$ ls -l /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240416
total 2528
-rw-r--r-- 1 shelley.melchior vpppg 2585806 Apr 17 14:31 evs.stats.narremean.mean.grid2obs.v20240416.stat
```

COMOUTsmall=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/atmos.20240416/narre/grid2obs`
Everything in COMOUTsmall has the original (older) timestamp.

Please confirm. I think Part 1, Scenario 4 looks good.

BinbinZhou-NOAA commented:
Shelley,

  My connection to dogwood was suddenly lost and can not be  connected
again.
After it returns, I'll check for you.

Binbin

On Wed, Apr 17, 2024 at 10:38 AM Shelley Melchior ***@***.***>
wrote:

> *Part 1, Scenario 4*
>
> For this test I am using VDATE=20240416.
> I deleted the final stat file in COMOUTfinal.
>
> logfile: /lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142666347
>
> The final stat file was rewritten (later time stamp).
>
> $ ls -l /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240416
> total 2528
> -rw-r--r-- 1 shelley.melchior vpppg 2585806 Apr 17 14:31 evs.stats.narremean.mean.grid2obs.v20240416.stat
>
> COMOUTsmall=
> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/atmos.20240416/narre/grid2obs
> Everything in COMOUTsmall has the original (older) timestamp.
>
> Please confirm. I think Part 1, Scenario 4 looks good.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2061418946>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFIVGNYNEQWTKQLUGV3Y52COLAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRRGQYTQOJUGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA commented:
Shelley,

  Dogwood has returned.
I have checked all files in $COMOUTfinal and $COMOUTsmall directories (with
VDATE=20240416).
All files are good.

Thanks!

Binbin

On Wed, Apr 17, 2024 at 10:38 AM Shelley Melchior ***@***.***>
wrote:

> *Part 1, Scenario 4*
>
> For this test I am using VDATE=20240416.
> I deleted the final stat file in COMOUTfinal.
>
> logfile: /lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142666347
>
> The final stat file was rewritten (later time stamp).
>
> $ ls -l /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/narre.20240416
> total 2528
> -rw-r--r-- 1 shelley.melchior vpppg 2585806 Apr 17 14:31 evs.stats.narremean.mean.grid2obs.v20240416.stat
>
> COMOUTsmall=
> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/narre/atmos.20240416/narre/grid2obs
> Everything in COMOUTsmall has the original (older) timestamp.
>
> Please confirm. I think Part 1, Scenario 4 looks good.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2061418946>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFIVGNYNEQWTKQLUGV3Y52COLAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRRGQYTQOJUGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Thanks for the confirmation, Binbin! Reading the description for Part 1, Scenario 5, this seems just like a brand new, or fresh, run, right? If there is no COMOUTsmall or COMOUTfinal, that is the same as a new run to me.

I do want to do one more test for stats, however. I will start a new run, for a different VDATE that I haven't run yet. I will let it run for 2 or 3 minutes, and then I will qdel the process. And then I will restart it. I will share the two log files for your review.

BinbinZhou-NOAA commented:
Shelley,

   OK, thanks!

Binbin

On Wed, Apr 17, 2024 at 12:47 PM Shelley Melchior ***@***.***>
wrote:

> Thanks for the confirmation, Binbin! Reading the description for Part 1,
> Scenario 5, this seems just like a brand new, or fresh, run, right? If
> there is no COMOUTsmall or COMOUTfinal, that is the same as a new run to me.
>
> I do want to do one more test for stats, however. I will start a new run,
> for a different VDATE that I haven't run yet. I will let it run for 2 or 3
> minutes, and then I will qdel the process. And then I will restart it. I
> will share the two log files for your review.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2061745004>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFISZ3DNLPTUQMZKFDLY52RR7AVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRRG42DKMBQGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Run 1 for VDATE=20240414
```
142675982.dbqs01               shelley.melchi* dev             jevs_narre_sta*     8514    1    16  500gb 00:45 R 00:01
```
logfile 1: `/lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142675982`
I ran qdel at 3 minutes into run time. `$ qdel 142675982.dbqs01`
`-rw-r--r-- 1 shelley.melchior emc 1101174 Apr 17 16:52 jevs_narre_stats.o142675982`

Run 2 for VDATE=20240414
```
142676298.dbqs01               shelley.melchi* dev             jevs_narre_sta*    53840    1    16  500gb 00:45 R 00:15
```
logfile 2: `/lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142676298`
COMOUTsmall and COMOUTfinal have all expected files.
The job completed in 21 minutes.

Now that I see this in action, I think this test is effectively your scenario 5! If I had run `qdel` a little later, it would be your scenario 3. Thank you for spelling out all the varieties for easy testing.

BinbinZhou-NOAA commented:
Shelley,

  I checked the 2 log files and all look good.

Binbin

On Wed, Apr 17, 2024 at 1:17 PM Shelley Melchior ***@***.***>
wrote:

> Run 1 for VDATE=20240414
>
> 142675982.dbqs01               shelley.melchi* dev             jevs_narre_sta*     8514    1    16  500gb 00:45 R 00:01
>
> logfile 1: /lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142675982
> I ran qdel at 3 minutes into run time. $ qdel 142675982.dbqs01
> -rw-r--r-- 1 shelley.melchior emc 1101174 Apr 17 16:52
> jevs_narre_stats.o142675982
>
> Run 2 for VDATE=20240414
>
> 142676298.dbqs01               shelley.melchi* dev             jevs_narre_sta*    53840    1    16  500gb 00:45 R 00:15
>
> logfile 2: /lfs/h2/emc/ptmp/shelley.melchior/jevs_narre_stats.o142676298
> COMOUTsmall and COMOUTfinal have all expected files.
> The job completed in 21 minutes.
>
> Now that I see this in action, I think this test is effectively your
> scenario 5! If I had run qdel a little later, it would be your scenario
> 3. Thank you for spelling out all the varieties for easy testing.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2061802090>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJVAPAUZR4LPQ7GIK3Y52VARAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRRHAYDEMBZGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Great! I will move onto plots restart testing.

ShelleyMelchior-NOAA commented:
**Part 2, Scenario 1**

This is the baseline. A complete run, end to end, for past 31 days and past 90 days.
logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past31days_plots.o142678071`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past90days_plots.o142678003`
COMOUT=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240416`
NOTE: as of hitting "send" on this comment (2:30p), the 90days plots run was still finishing up. It is at 57 minutes of run time. It will conclude soon.

BinbinZhou-NOAA commented:
Shelley,

  Both 31 and 90-day plot files are good, and all of the  files in the
restart directories
are correct.

Binbin


On Wed, Apr 17, 2024 at 2:30 PM Shelley Melchior ***@***.***>
wrote:

> *Part 2, Scenario 1*
>
> This is the baseline. A complete run, end to end, for past 31 days and
> past 90 days.
> logfiles:
>
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past31days_plots.o142678071
>
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past90days_plots.o142678003
> COMOUT=
> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240416
> NOTE: as of hitting "send" on this comment (2:30p), the 90days plots run
> was still finishing up. It is at 57 minutes of run time. It will conclude
> soon.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2061952185>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFOR64RRVQGUZMEFQRTY525S3AVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRRHE2TEMJYGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
**Part 2, Scenario 2**

Deleted the vis `*.png` and VIS `*.completed` files from the restart dirs for `31/` and `90/`.

logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past31days_plots.o142690676`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past90days_plots.o142690663`

COMOUT restart dirs have newly created vis and VIS files.
`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240416/restart/31`
`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240416/restart/90`

The final tar files were regenerated w/ a later time stamp.
`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240416`

BinbinZhou-NOAA commented:
Shelley,

  For scenario 2, the final tar files are good, and all files in the
restart directory
are good as well.

Thanks!

Binbin

On Wed, Apr 17, 2024 at 5:46 PM Shelley Melchior ***@***.***>
wrote:

> *Part 2, Scenario 2*
>
> Deleted the vis *.png and VIS *.completed files from the restart dirs for
> 31/ and 90/.
>
> logfiles:
>
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past31days_plots.o142690676
>
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past90days_plots.o142690663
>
> COMOUT restart dirs have newly created vis and VIS files.
>
> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240416/restart/31
>
> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240416/restart/90
>
> The final tar files were regenerated w/ a later time stamp.
> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240416
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2062462599>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFIDOLVQIQFP6624URDY53US7AVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRSGQ3DENJZHE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Excellent -- thank you for the confirmation, @BinbinZhou-NOAA . I will do one final test, which is to start the job for a specific VDATE, then stop it w/ the qdel command a handful of minutes into the processing. And then restart the job. I will update the PR w/ the results.

ShelleyMelchior-NOAA commented:
For completeness, I will also run narre stats and plots, end to end, w/ the updated beta versions for met/metplus before merging.

BinbinZhou-NOAA commented:
Shelley,

  Great! Thanks! for coordinating all of these! After this PR is completed,
I'll submit a new PR for  sref restart. The testing procedure of the sref
restart is similar to that of narre restart.

Binbin



On Thu, Apr 18, 2024 at 8:29 AM Shelley Melchior ***@***.***>
wrote:

> For completeness, I will also run narre stats and plots, end to end, w/
> the updated beta versions for met/metplus before merging.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2063753021>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFMD6R7AZZXVOLE7PX3Y564DTAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRTG42TGMBSGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Sounds great, Binbin. Can you also please update this PR branch w/ a sync fork?

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into narre_restart

BinbinZhou-NOAA commented:
Shelley,

  Sure, I have updated it.

Binbin

On Thu, Apr 18, 2024 at 9:54 AM Shelley Melchior ***@***.***>
wrote:

> Sounds great, Binbin. Can you also please update this PR branch w/ a sync
> fork?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2063924798>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFKJBZHZBEUODKC4CCDY57GBNAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRTHEZDINZZHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
**Final restart test for plots**

**Run 1**
logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past31days_plots.o142745924`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past90days_plots.o142745923`

```
142745923.dbqs01               shelley.melchi* dev             jevs_narre_pas*    27301    1     8    8gb 01:10 R 00:06
142745924.dbqs01               shelley.melchi* dev             jevs_narre_pas*   166984    1     8    8gb 00:30 R 00:06
```
I qdel'd the processes after a few minutes. `qdel 142745923.dbqs01 142745924.dbqs01`
31 days had hgt PDs completed. 90 days didn't have anything posted to the restart dir yet/
```
[dlogin09 /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240417/restart]$ ls -l 31
total 780
-rw-r--r-- 1 shelley.melchior emc 397456 Apr 18 14:10 performance_diagram_regional_g130_valid_
00z_03z_06z_09z_12z_15z_18z_21z_hgt_f1_to_f12__lt152lt305lt914lt1524lt3048.png
-rw-r--r-- 1 shelley.melchior emc 396035 Apr 18 14:10 performance_diagram_regional_g242_valid_
00z_03z_06z_09z_12z_15z_18z_21z_hgt_f1_to_f12__lt152lt305lt914lt1524lt3048.png
-rw-r--r-- 1 shelley.melchior emc      0 Apr 18 14:10 run_narre_G130.performance_diagram.HGTcldceil.ctc.completed
-rw-r--r-- 1 shelley.melchior emc      0 Apr 18 14:10 run_narre_G242.performance_diagram.HGTcldceil.ctc.completed
[dlogin09 /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240417/restart]$ ls 90
[dlogin09 /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240417/restart]$ 
```

**Run 2**
logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past31days_plots.o142748214`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past90days_plots.o142748212`
COMOUT=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240417`

BinbinZhou-NOAA commented:
Shelley,

  For run1, by checking the 31 and 90-day o* files, I can see that before
the jobs were manually killed,  the HGTcldceil (ceiling) tasks were
completed and their *.completed and *.png files were generated in the
restart directory
  For run2, I can see that the existing HGTcldceil png files were copied
from the restart directory generated from run1 to the working directories
and final tar files for both 31 and 90-day are generated with good shape.

Thanks!

Binbin



On Thu, Apr 18, 2024 at 12:34 PM Shelley Melchior ***@***.***>
wrote:

> *Final restart test for plots*
>
> *Run 1*
> logfiles:
>
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past31days_plots.o142745924
>
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past90days_plots.o142745923
>
> 142745923.dbqs01               shelley.melchi* dev             jevs_narre_pas*    27301    1     8    8gb 01:10 R 00:06
> 142745924.dbqs01               shelley.melchi* dev             jevs_narre_pas*   166984    1     8    8gb 00:30 R 00:06
>
> I qdel'd the processes after a few minutes. qdel 142745923.dbqs01
> 142745924.dbqs01
> 31 days had hgt PDs completed. 90 days didn't have anything posted to the
> restart dir yet/
>
> [dlogin09 /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240417/restart]$ ls -l 31
> total 780
> -rw-r--r-- 1 shelley.melchior emc 397456 Apr 18 14:10 performance_diagram_regional_g130_valid_
> 00z_03z_06z_09z_12z_15z_18z_21z_hgt_f1_to_f12__lt152lt305lt914lt1524lt3048.png
> -rw-r--r-- 1 shelley.melchior emc 396035 Apr 18 14:10 performance_diagram_regional_g242_valid_
> 00z_03z_06z_09z_12z_15z_18z_21z_hgt_f1_to_f12__lt152lt305lt914lt1524lt3048.png
> -rw-r--r-- 1 shelley.melchior emc      0 Apr 18 14:10 run_narre_G130.performance_diagram.HGTcldceil.ctc.completed
> -rw-r--r-- 1 shelley.melchior emc      0 Apr 18 14:10 run_narre_G242.performance_diagram.HGTcldceil.ctc.completed
> [dlogin09 /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240417/restart]$ ls 90
> [dlogin09 /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240417/restart]$
>
> *Run 2*
> logfiles:
>
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past31days_plots.o142748214
>
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past90days_plots.o142748212
> COMOUT=
> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240417
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2064477682>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFP75HV36UD2R6AFAHTY57Y23AVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRUGQ3TONRYGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA commented:
Correction:

  In run1, restart files were not generated, so the its run2,  the job
started from
from the beginning, and the tar file was correctly generated.

Binbin

On Thu, Apr 18, 2024 at 1:51 PM Binbin Zhou - NOAA Affiliate <
***@***.***> wrote:

> Shelley,
>
>   For run1, by checking the 31 and 90-day o* files, I can see that before
> the jobs were manually killed,  the HGTcldceil (ceiling) tasks were
> completed and their *.completed and *.png files were generated in the
> restart directory
>   For run2, I can see that the existing HGTcldceil png files were copied
> from the restart directory generated from run1 to the working directories
> and final tar files for both 31 and 90-day are generated with good shape.
>
> Thanks!
>
> Binbin
>
>
>
> On Thu, Apr 18, 2024 at 12:34 PM Shelley Melchior <
> ***@***.***> wrote:
>
>> *Final restart test for plots*
>>
>> *Run 1*
>> logfiles:
>>
>> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past31days_plots.o142745924
>>
>> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past90days_plots.o142745923
>>
>> 142745923.dbqs01               shelley.melchi* dev             jevs_narre_pas*    27301    1     8    8gb 01:10 R 00:06
>> 142745924.dbqs01               shelley.melchi* dev             jevs_narre_pas*   166984    1     8    8gb 00:30 R 00:06
>>
>> I qdel'd the processes after a few minutes. qdel 142745923.dbqs01
>> 142745924.dbqs01
>> 31 days had hgt PDs completed. 90 days didn't have anything posted to the
>> restart dir yet/
>>
>> [dlogin09 /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240417/restart]$ ls -l 31
>> total 780
>> -rw-r--r-- 1 shelley.melchior emc 397456 Apr 18 14:10 performance_diagram_regional_g130_valid_
>> 00z_03z_06z_09z_12z_15z_18z_21z_hgt_f1_to_f12__lt152lt305lt914lt1524lt3048.png
>> -rw-r--r-- 1 shelley.melchior emc 396035 Apr 18 14:10 performance_diagram_regional_g242_valid_
>> 00z_03z_06z_09z_12z_15z_18z_21z_hgt_f1_to_f12__lt152lt305lt914lt1524lt3048.png
>> -rw-r--r-- 1 shelley.melchior emc      0 Apr 18 14:10 run_narre_G130.performance_diagram.HGTcldceil.ctc.completed
>> -rw-r--r-- 1 shelley.melchior emc      0 Apr 18 14:10 run_narre_G242.performance_diagram.HGTcldceil.ctc.completed
>> [dlogin09 /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240417/restart]$ ls 90
>> [dlogin09 /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240417/restart]$
>>
>> *Run 2*
>> logfiles:
>>
>> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past31days_plots.o142748214
>>
>> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_past90days_plots.o142748212
>> COMOUT=
>> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20240417
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2064477682>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQBMPFP75HV36UD2R6AFAHTY57Y23AVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRUGQ3TONRYGI>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
>
> Binbin Zhou
>
> Physical Scientist
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Great! I think we are all square w/ restart testing for both stats and plots. Next I will run stats and plots w/ the MET/METplus beta3 versions. I will not test restart, but rather end-to-end runs.

I have already completed **stats** for your review.

logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_stats.o142765583`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v2.0/stats/narre`

BinbinZhou-NOAA commented:
Shelley,

  The stats run is fine with the correct stat file and the restart files.

Binbin

On Thu, Apr 18, 2024 at 3:53 PM Shelley Melchior ***@***.***>
wrote:

> Great! I think we are all square w/ restart testing for both stats and
> plots. Next I will run stats and plots w/ the MET/METplus beta3 versions. I
> will not test restart, but rather end-to-end runs.
>
> I have already completed *stats* for your review.
>
> logfile:
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_narre_stats.o142765583
> COMOUT=/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v2.0/stats/narre
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2065140188>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFLPSNFW57FSKVBNSHDY6AQBZAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRVGE2DAMJYHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Awesome! I will run a few more days of stats w/ the new MET/METplus versions. And then I will run plots using those new stats. Will your plots work OK w/ only a few days of stats available?

BinbinZhou-NOAA commented:
Shelley,

  Yes, the plotting scripts work well with only a few days of stat files

Binbin

On Thu, Apr 18, 2024 at 4:13 PM Shelley Melchior ***@***.***>
wrote:

> Awesome! I will run a few more days of stats w/ the new MET/METplus
> versions. And then I will run plots using those new stats. Will your plots
> work OK w/ only a few days of stats available?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2065208385>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFN6KTI67QRTIRQUOADY6ASP7AVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRVGIYDQMZYGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
HI @BinbinZhou-NOAA , I just want to update you that I am currently w/o a working CAC. My CAC renewal yesterday has yielded a new CAC that is not permitting VPN (or GFE login) access. Once that is smoothed out I can continue on the PR.

BinbinZhou-NOAA commented:
Shelley,

  Thanks for the update.

Binbin

On Fri, Apr 19, 2024 at 12:29 PM Shelley Melchior ***@***.***>
wrote:

> HI @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , I just want to
> update you that I am currently w/o a working CAC. My CAC renewal yesterday
> has yielded a new CAC that is not permitting VPN (or GFE login) access.
> Once that is smoothed out I can continue on the PR.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2066909684>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFIYDR63SJT3E2IT7TDY6FA7ZAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRWHEYDSNRYGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
@BinbinZhou-NOAA , I am back in business. My CAC is working and I can get onto VPN and WCOSS2.

I have generated stats for VDATES 20240416 - 20240419 using the MET/METplus beta3 versions.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v2.0/stats/narre`

I then ran the plots processing using these stats, to make sure the plots processing works OK w/ the MET/METplus beta3 versions.
COMOUT=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v2.0/plots/narre/atmos.20240419/`

Please review.

BinbinZhou-NOAA commented:
Shelley,

   Both stat file and plot tar files are good. Restart files are also good.

Binbin

On Sat, Apr 20, 2024 at 12:24 PM Shelley Melchior ***@***.***>
wrote:

> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , I am back in
> business. My CAC is working and I can get onto VPN and WCOSS2.
>
> I have generated stats for VDATES 20240416 - 20240419 using the
> MET/METplus beta3 versions.
> COMOUT=/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v2.0/stats/narre
>
> I then ran the plots processing using these stats, to make sure the plots
> processing works OK w/ the MET/METplus beta3 versions.
> COMOUT=
> /lfs/h2/emc/ptmp/shelley.melchior/evs/v2.0/plots/narre/atmos.20240419/
>
> Please review.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2067721514>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFODPJVNPXFSFJ3Q6YLY6KJE7AVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRXG4ZDCNJRGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Excellent! Please sync your fork and then we will be ready to merge this PR.

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into narre_restart

BinbinZhou-NOAA commented:
Shelley,

   Sure, just did it.

Binbin

On Sun, Apr 21, 2024 at 9:16 AM Shelley Melchior ***@***.***>
wrote:

> Excellent! Please sync your fork and then we will be ready to merge this
> PR.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#issuecomment-2068039937>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFMDKAK6RROW5UYMOA3Y6O3ZDAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANRYGAZTSOJTG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA reviewed (approved):
The updates to NARRE to add restart work as expected. This PR is approved for merge.

BinbinZhou-NOAA commented:
Shelley,

  Thanks!

Binbin

On Sun, Apr 21, 2024 at 9:27 AM Shelley Melchior ***@***.***>
wrote:

> Merged #465 <https://github.com/NOAA-EMC/EVS/pull/465> into develop.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/465#event-12550890255>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJCKVOPVRSQFEI7VC3Y6O5CZAVCNFSM6AAAAABFZMM35OVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJSGU2TAOBZGAZDKNI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683
